src/index.html

```diff
@@ -4,9 +4,404 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        class="header__logo"
+        href="#home" 
+      >
```

Comment:
```suggestion
      <a class="header__logo" href="#home" >
```
if there are less than 3 attributes there is no need to put them on the new line

Reply:
ok. as you wish. next time I will write all three on one line))